### DISCUSSION TITLE
Qfield recommended compatible devices

### DISCUSSION THREAD

**Discussion by dukytony** [opengisch/QField#4005] [Q&A]
Hello,

We are currently looking to invest into field devices for qfield use on field.

So far we ve been using Samsung galaxy A7 LTE tablets for average/good results.

Can you guys recommend a tested compatible device?

Should we go for "pro" type devices like Samsung Galaxy Tab Active Pro ?

We are especially looking at devices with high quality 4 sensor as most of our database/layers are hosted on a postgresql server.

Can you recommend a brand in particular?

Thank you very much for your help!

**Comment by meyerlor** [comment]
THat would be very interesting - especially with the camra-crash bug still around, it would be interesting to know, on which devices it works flawlessly.
Samsung Galaxy Tab A 10.5 can be ruled out, it is quite slow with the new QField versions and the camera crashes frequently (Nativ/Not-Native one)

**Comment by soester** [comment]
We are working with the following devices in production, without camera issues:
- Samsung Galaxy Tab Active2 (8.0, LTE) (SM-T395N)
- Samsung Galaxy Tab Active3

Tab Active2 is slower than Tab Active3 (less ram, slower processor) and is only updated to Android 8. Tab Active 3 is updated to Android 10.

Perhaps the findings could be compiled somewhere in a table (e.g. Github Discussion or QField Docs)?.
Everybody will need to update their devices at some point and therefore be interested in first hand experiences.

**Comment by meyerlor** [comment]
Samsung Galaxy Tab S8 (SM-X700) works very well

**Reply by Ipickedausername** [reply]
So it is with the Samsung galaxy 9
With room for improvement, as some mapping applications are running more smoothly on this same phone.
